addons/resource/models/res_company.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    def _get_default_resource_calendar_ids(self):
+        return [(0, 0, {'name': 'Standard 40 hours/week'})]
+
+    resource_calendar_ids = fields.One2many(
+        'resource.calendar', 'company_id', 'Working Hours',
+        default=_get_default_resource_calendar_ids)
```

Comment:
No point in having a method for that, it won't be anything, so this could just as well be the value directly.